ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java

```diff
@@ -290,6 +290,7 @@ private void setIconSource(ReadableMap source, IconControllerListener controller
               .setOldController(holder.getController())
               .build();
       holder.setController(controller);
+      holder.getTopLevelDrawable().setVisible(true, true);
```

Comment:
> What is true, true?

Sorry for not being clear. I meant there should be a comment in the code explaining this.